chat.py

```diff
@@ -70,16 +80,71 @@ def export(db_path: str, output_dir: str = None):
         logger.error(error_message)
         raise typer.Exit(code=1)
 
+def get_auto_db_path(latest: bool) -> str:
+    system = platform.system()
+    home = Path.home()
+
+    if system == "Windows":
+        base_path = Path(os.environ.get("APPDATA")) / "Cursor" / "User" / "workspaceStorage"
+    elif system == "Darwin":  # macOS
+        base_path = home / "Library" / "Application Support" / "Cursor" / "User" / "workspaceStorage"
+    elif system == "Linux":
+        base_path = home / ".config" / "Cursor" / "User" / "workspaceStorage"
+    else:
+        raise ValueError(f"Unsupported operating system: {system}")
+
+    if not base_path.exists():
+        raise FileNotFoundError(f"Cursor workspace storage directory not found: {base_path}")
+
+    if latest:
+        workspace_folder = max(base_path.glob("*"), key=os.path.getmtime)
+    else:
+        workspace_folders = list(base_path.glob("*"))
+        if not workspace_folders:
+            raise FileNotFoundError(f"No workspace folders found in {base_path}")
+        workspace_folder = workspace_folders[0]
+
+    db_path = workspace_folder / "state.vscdb"
+    if not db_path.exists():
+        raise FileNotFoundError(f"state.vscdb not found in {workspace_folder}")
+
+    return str(db_path)
+
+def get_auto_base_path() -> str:
+    system = platform.system()
+    home = Path.home()
+
+    if system == "Windows":
+        base_path = Path(os.environ.get("APPDATA")) / "Cursor" / "User" / "workspaceStorage"
+    elif system == "Darwin":  # macOS
+        base_path = home / "Library" / "Application Support" / "Cursor" / "User" / "workspaceStorage"
+    elif system == "Linux":
+        base_path = home / ".config" / "Cursor" / "User" / "workspaceStorage"
```

Comment:
This is duplicate like this, let's refactor getting the default path into a separate function which can be then used by both `get_auto_db_path` and `get_auto_base_path`.